### DISCUSSION TITLE
What will ultimately come out of this repository?

### DISCUSSION THREAD

**Discussion by yellowmomo** [HwangDoHyun28/github_manager#4] [Q&A] [answered]
The same question as the title.
I understood the intention of getting used to the various functions of github.
However, I think not only getting used to it, but ultimately the direction of completing something will be a more productive activity.
What shape do you expect it to be if something were to produce results?

**Comment by HwangDoHyun28** [comment, marked as answer]
That's a very good question.
Please understand that I have no choice but to give you a limited answer because I can only use githubaction.
Githubaction was understood to be a useful tool, such as a pipeline, that intertwines a number of functions into a large function.
I'm trying to use github action to automatically make my github more manageable than me even if I don't do something passively in the future.
Please wait and see if that's possible!
